topics/01_General-and-Getting-Started/~Getting_Started.md

```diff
@@ -8,25 +8,237 @@
 |metadata|
 -->
 
-# Get Started with Ignite UI
-There are a number of different ways to get started building desktop and mobile applications with Ignite UI&trade;. Check out the links below for ways for you to quickly get started building your app!
+# In this topic
 
-## Install, Download or Reference Files
-You have a number of different options available to you when it comes to getting access to Ignite UI files:
+This topic contains the following sections:
 
-- **[Platform Installer](https://www.infragistics.com/my-account/keys-and-downloads)**: Licensed customers can login and download the platform installer.
+-   [Introduction](#introduction)
+-   [Download and Install](#download)
+-   [Host Ignite UI in your project](#hosting)
+    -   [Using NPM, JSPM, NuGet](#package_managers)
+    -   [Add CSS and JavaScript references](#add_references)
+    -   [Sample Ignite UI Boilerplate HTML page (using trial CDN links)](#boilerplate)
+-   [Add your first control](#first_control)
+    -   [Add an igGrid Directly](#directly)
+    -   [Add an igGrid using Page Designer](#page_designer)
+-   [Get just what you need](#custom_download)
+-   [Using CDN Links](#cdn)
+-   [TypeScript Directives](#typescript)
+-   [AngularJS Extensions](#angularjs)
+-   [Angular 2 Extensions](#angular)
+-   [ReactJS Extensions](#reactjs)
+-   [Related Content](#related_content)
 
-- **[Download (full product or custom build)](https://www.igniteui.com/download)**: Download the full version of Ignite UI or create a custom build to only include the CSS and JavaScript related to specific controls.
+# <a id="introduction"></a>Introduction
 
-- **[CDN Links](https://www.igniteui.com/download)**: From the downloads page you also have access to the Ignite UI files on the Infragistics CDN for bundled versions of Ignite UI.  
- 
-> **Note**: For the download and CDN options, if you log in as an existing customer you are able to access production-ready files otherwise you'll get trial versions of Ignite UI.
+Ignite UI&trade; is an advanced HTML5+ toolset that helps you create stunning, modern Web apps. Building on jQuery and jQuery UI, it primarily consists of feature rich, high-performing UI controls/widgets such as all kinds of charts, data visualization maps, (hierarchical, editable) data grids, pivot grids, enhanced editors (combo box, masked editors, HTML editor, date picker, to name a few), flexible data source connectors, and a whole lot more.
 
-## Jump Right In
-If you'd like to dive right in by working with the controls directly, the [Page Designer allows you to work with each control in a WYSIWYG environment](http://designer.igniteui.com/). The designer comes is complete with sample data that makes it easy to get started without having to go through any sort of setup process.
+Ignite UI comes in two vesions: 
+-   Full - a paid version that contains the complete toolset.
```

Comment:
Maybe switch them around - go from OSS to Full, rather than the other way around.